ledger/sandbox/src/main/resources/db/migration/h2database/V34__sequential_row_id.sql

```diff
@@ -0,0 +1,23 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V34: Add a sequential row_id to participant_events
+--
+---------------------------------------------------------------------------------------------------
+
+-- 1. add the column
+alter table participant_events add column row_id bigint auto_increment;
+
+
+-- 2. fix the row_id to be sequential according to the order of (event_offset, transaction_id, node_index)
+-- don't need to migrate H2 datasets
+
+-- 3. drop the now unused index
+-- looks like we never created this index in H2
```

Comment:
yeah my bad, I added the name to the index and dropping it by name now? Or should I just remove it from the original script? Why do we even bother supporting flyway migration scripts for H2?

Reply:
We are using schema migration only, but indeed it may be the case that it could make more sense to simply keep a squashed schema. This could be part of the Sandbox/DAML-on-SQL split.

Reply:
ok... in any case, I will not be squashing it as part of this PR.